parsely_raw_data/schema.py

```diff
@@ -21,7 +19,7 @@
 e.g. `campaign_id` will be null for events where campaign is false.
 """
 SCHEMA = [
-    {"key": "action", "ex": "pageview", "type": str, "size": 256, "req": True},
+    {"key": "action", "ex": "pageview", "type": str, "size": 256},
```

Comment:
I did not know the action field could be empty

Reply:
it shouldn't be empty, and i think this was from years ago where there were null actions in DPL, and it was causing havoc in the Redshift create table statement and parsing a rare null action. It's not something that should be upheld in the schema, and should instead be handled down the ETL pipeline.